src/targets/nuget.ts

```diff
@@ -17,6 +17,12 @@ export const DEFAULT_NUGET_SERVER_URL = 'https://api.nuget.org/v3/index.json';
 const DEFAULT_NUGET_REGEX = /^.*\d\.\d.*\.nupkg$/;
 const SYMBOLS_NUGET_REGEX = /^.*\d\.\d.*\.snupkg$/;
 
+/**
+ * Spawn options to run outside the repository folder to avoid global.json constraints
+ * (we don't need specific dotnet/workload versions just to upload to nuget)
+ */
+const SPAWN_OPTIONS = { cwd: '/' };
```

Comment:
question: Is this the right way to do it?

Alternative:
As `private readonly` field on `class NugetTarget`.